lib/address.js

```diff
@@ -505,3 +504,5 @@ Address.prototype.inspect = function() {
 };
 
 module.exports = Address;
+
+var Script = require('./script');
```

Comment:
Resolves the circular dependency between `address.js` and `script.js` without needing to have several `address.js` require statements in functions in `script.js` that slowed each function down each time it was invoked. Using a late definition here is faster.